topydo/ui/Main.py

```diff
@@ -95,6 +95,8 @@ def __init__(self):
         self.todofile = TodoFile.TodoFile(config().todotxt())
         self.todolist = TodoList.TodoList(self.todofile.read())
 
+        self.pending_todos = []
```

Comment:
Besides, if you make it a `set()`,  you don't need to worry whether a todo item was already present in the pending list (in the code below). As far as I can see, order of the todo items is not important in this context.